user/ip-addresses.md

```diff
@@ -11,8 +11,8 @@ on the infrastructure your builds are running on:
 
 | Infrastructure                  | IP ranges                                                                                                                                                                  |
 | ------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| Container-based (travis-ci.com) | `nat-com.aws-us-east-1.travisci.net` (`54.172.141.90/32` `52.3.133.20/32`) and `workers-nat-com-shared-2.aws-us-east-1.travisci.net` (`52.45.185.117/32` `52.54.31.11/32` `54.87.185.35/32` `54.82.137.203/32`) |
-| Container-based (travis-ci.org) | `nat-org.aws-us-east-1.travisci.net` (`52.0.240.122/32` `52.22.60.255/32`) and `workers-nat-org-shared-2.aws-us-east-1.travisci.net` (`52.45.220.64/32` `52.54.40.118/32` `54.89.89.104/32` `54.87.141.246/32`) |
+| Container-based (travis-ci.com) | `nat-com.aws-us-east-1.travisci.net` (`54.172.141.90/32` `52.3.133.20/32`) and `workers-nat-com-shared-2.aws-us-east-1.travisci.net` (`52.45.220.64/32` `52.54.40.118/32` `54.89.89.104/32` `54.82.137.203/32`) |
```

Comment:
Would it make sense to add a <div> tag before the _and.. _  to force a new line within the same row?